terraform/environment.tf

```diff
@@ -4,7 +4,6 @@ locals {
   is_dev              = !(local.is_prod || local.is_test)
   env_name            = local.is_prod ? "prod" : terraform.workspace
   env_letter          = upper(substr(local.env_name, 0, 1))
-  subscription_letter = local.is_prod ? "P" : "D"
```

Comment:
this is the same unused variable that was yanked in https://github.com/Office-of-Digital-Services/cdt-ods-disaster-recovery/pull/50